project1/client/input_parser/input_parser.cpp

```diff
@@ -0,0 +1,157 @@
+#include "input_parser.h"
+
+namespace client::input_parser {
+    using ftp_messages::Request;
+    using ftp_messages::GetRequest;
+    using ftp_messages::PutRequest;
+    using ftp_messages::ChangeDirRequest;
+    using ftp_messages::ListRequest;
+    using ftp_messages::PwdRequest;
+    using ftp_messages::DeleteRequest;
+    using ftp_messages::QuitRequest;
+    using ftp_messages::MakeDirRequest;
+
+    InputParser::InputParser(std::string &cmd) {
+        this->fn = "";
```

Comment:
(minor) These variable names aren't very descriptive. Also, is there a reason you have to prefix with `this->`?